ibc-clients/cw-context/src/types/helper.rs

```diff
@@ -0,0 +1,30 @@
+use ibc_client_wasm_types::{SUBJECT_PREFIX, SUBSTITUTE_PREFIX};
+
+/// The MigrationPrefix enumerates the prefix type used during migration mode.
+/// The migration mode is activated when there is an incoming
+/// `MigrateClientStore` message. It specifies the prefix key for either the
```

Comment:
We may link `MigrateClientStore` in docs.

Reply:
[6de21ed](https://github.com/cosmos/ibc-rs/pull/1167/commits/6de21edb62137b9d74daf1bcacf3b626e45013cc)